### DISCUSSION TITLE
Ask for object pascal inclusion in https://1brc.dev website

### DISCUSSION THREAD

**Discussion by synopse** [ObjectPascal-Community/1brc-ObjectPascal#40] [Ideas]
Perhaps we could ask to add an entry about object pascal in 
https://1brc.dev

It could be done even before finishing the challenge in May, just to increase visilibty.

**Comment by synopse** [comment]
Note that we can't compare the timing between languages easily.
The hardware is quite diverse, and even the input file seems not to match.

For instance, the measurements.txt file seems to be 12GB from https://github.com/dannyvankooten/1brc?tab=readme-ov-file#create-the-measurements-file-with-1b-rows
My guess is that they used LF for the line feed, whereas we used CR+LF.
And it seems they used only 400 station names, and we use more than 40,000 - some with huge names! It makes a huge difference about data locality and the hash table involved.

Difficult to compare things here.
My guess is that object pascal could be in the top list, way above Java.

**Reply by gcarreno** [reply]
I added the section "Differences From Original" exactly due to my decision to make it go way up to 11!

And I do advise that we do comparison within that context.

**Comment by gcarreno** [comment]
Okidokes, posted an issue on there to see if we can be listed.
